src/Module.zig

```diff
@@ -366,8 +368,8 @@ pub const Decl = struct {
     is_exported: bool,
     /// Whether the ZIR code provides an align instruction.
     has_align: bool,
-    /// Whether the ZIR code provides a linksection instruction.
-    has_linksection: bool,
+    /// Whether the ZIR code provides a linksection and address space instruction.
+    has_linksection_or_addrspace: bool,
```

Comment:
`addrspace` comes before `linksection`, so it could be called `has_addrspace_or_linksection` for consistency. Doesnt make a notable difference though, so just close if you disagree.

Reply:
Hm yes, though this would have some further reaching consequences than just the name (like the zir format). I think this is probably also fine